projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -71,7 +72,65 @@ context('Assisted Service Module', () => {
       cy.log(
         '--> Stop customer emulation using the end session button in the ASM UI'
       );
-      cy.get('cx-customer-emulation button').click();
+      cy.get('cx-customer-emulation [data-cy=logout]').click();
```

Comment:
As a convention, we shouldn't add attributes in the dom like data-cy=logout for the sole purpose of tests.
DOM elements retrieved with css selectors.